addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -343,9 +323,8 @@ def migration_invoice_moves(env):
             SET exclude_from_invoice_tab = TRUE
             FROM account_move_line aml2
```

Comment:
No need here of the `FROM` part and you can optimize for not re-writing some lines if not needed:
```python
            """
            UPDATE account_move_line aml
            SET exclude_from_invoice_tab = TRUE
            WHERE aml.old_invoice_line_id IS NULL
                AND NOT exclude_from_invoice_tab"""
```

Reply:
maybe should say `NOT aml.exclude_from_invoice_tab or aml.exclude_from_invoice_tab IS NULL`

Reply:
A boolean field can have `NULL`?

Reply:
if not filled?

Reply:
OK, just remove then that condition.
```python
            """
            UPDATE account_move_line aml
            SET exclude_from_invoice_tab = TRUE
            WHERE aml.old_invoice_line_id IS NULL
            """
```

Reply:
done